src/feature/wines/detail/components/WineDetailCardSection.tsx

```
@@ -2,14 +2,14 @@
 
 import DetailCard from '@/feature/wines/components/card/DetailCard';
 import { useWineDetail } from '@/feature/wines/hooks/useWineDetailsQuery';
-
+import Error from '@/shared/components/common/error';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)